src/SOS/Strike/strike.cpp

```diff
@@ -9459,85 +9690,52 @@ DECLARE_API(u)
         {
             return E_OUTOFMEMORY;
         }
-        
+
         GCInfoToken gcInfoToken = { table, GCINFO_VERSION };
         g_targetMachine->DumpGCInfo(gcInfoToken, methodSize, DecodeGCTableEntry, false /*encBytes*/, false /*bPrintHeader*/);
-    }    
+    }
+    return S_OK;
+}
 
-    SOSEHInfo *pInfo = NULL;
-    if (fWithEHInfo)
+HRESULT displayIntermediateLanguage(BOOL bIL, const DacpCodeHeaderData& codeHeaderData,
```

Comment:
We should simply remove this function when the interleaving is done.

Reply:
@cshung the naming of the function is confusing but this also populates the map. We can rename this and remove the display statements for dumping the map or better add behind a flag for debugging. What do you say about that?

Reply:
Of course, I forgot. Sure, rename sounds fine to me.
A flag for debugging sounds nice too.